src/burner/sdl/main.cpp

```diff
@@ -275,6 +275,7 @@ int main(int argc, char *argv[])
       printf("e.g.: %s -menu -joy\n", argv[0]);
       printf("For NeoCD games:\n");
       printf("%s neocdz -cd path/to/ccd/filename.ccd\n", argv[0]);
+      printf("Usage is resitricted by the licence at https://raw.githubusercontent.com/finalburnneo/FBNeo/master/src/license.txt\n");
```

Comment:
kev, some bad news re: ccd - a lot of the subchannel data for ccd is bad and causes skips in audio etc.  It's probably best to prefer/recommend using bin/cue (although still support .ccd behind the scenes)
Unfortunately I found this out _after_ putting so much time into supporting it properly, as well as researched the issue, did tests in other emus and talked with other devs about it.  luckily bin/cue is 100% solid, though :)